### PR TITLE
core[patch]: simple fallback streaming

### PR DESCRIPTION
**Pull request by baskaryan**

### CODE REVIEW

File: libs/core/langchain_core/runnables/fallbacks.py [hunk 1]
```diff
@@ -343,3 +347,74 @@ async def abatch(
...
         await asyncio.gather(*(rm.on_chain_error(first_error) for rm in run_managers))
         raise first_error
+
+    def stream(
+        self,
+        input: Input,
+        config: Optional[RunnableConfig] = None,
+        **kwargs: Optional[Any],
+    ) -> Iterator[Output]:
+        # setup callbacks
```
Comment (eyurtsev) [inline libs/core/langchain_core/runnables/fallbacks.py:357]: Would you mind adding a doc-string here to document the semantics a bit better; i.e., what happens if the exception it not during the first chunk?

File: libs/core/langchain_core/runnables/fallbacks.py [hunk 2]
```diff
@@ -343,3 +347,74 @@ async def abatch(
...
+        for runnable in self.runnables:
+            try:
+                stream = runnable.stream(
+                    input,
+                    patch_config(config, callbacks=run_manager.get_child()),
+                    **kwargs,
+                )
+                chunk = next(stream)
+            except self.exceptions_to_handle as e:
+                first_error = e if first_error is None else first_error
```
Comment (eyurtsev) [inline libs/core/langchain_core/runnables/fallbacks.py:374]: We have an `on_retry` callback. I was expecting to see it used within this Runnable, but don't see it anywhere. Do we need to add it to the various calls or do we think it doesn't make sense for this runnable or some other issue?

**Review by eyurtsev — APPROVED** [review]
Looks good to me

**Comment by baskaryan** [comment]
still need to add passing in exception

**Pull request merged**